CoordgenMinimizer.cpp

```diff
@@ -72,7 +74,18 @@ void CoordgenMinimizer::run()
     }
 
     for (int iterations = 0; iterations < m_maxIterations; ++iterations) {
-        scoreInteractions();
+        float energy = scoreInteractions();
+
+        // #ifdef ...
```

Comment:
Let's call it DEBUG_MINIMIZATION_COORDINATES and have it cover all of this. You can enable defines in the cmake command `-DDEBUG_MINIMIZATION_COORDINATES=ON`.

Reply:
I tried adding this but the file writing doesn't occur after running this command and rebuilding. I just updated this PR -- do you know what I'm missing?

Reply:
Like, you reran cmake and make install? huh. 

Did the `make install` step build anything? I wonder if you need a rebuild after updating the macro

Reply:
Yeah, that wasn't working either, I even tried a clean build